openshift_scalability/config/golang/nodeVertical-labeled-nodes.yaml

```diff
@@ -0,0 +1,21 @@
+provider: local
+ClusterLoader:
+  cleanup: true
```

Comment:
can we set cleanup to false ? So we can keep the pods deployed for 10 minutes before the project clusterproject is terminated.

Reply:
I think introducing a sleep param in clusterloader which when defined should sleep before deleting the namespace is the way to go. What do you think?